### PR TITLE
Feature/Display ROI on Coronal and Sagittal Views

### PR DESCRIPTION
**Pull request by tdlinh2712**
This PR allows ROI to be displayed on coronal and sagittal views. To do this, we have made the following changes to the program:
- Refactor DICOM View - breaking DICOM View class into three subclasses, named DICOMAxialView, DICOMCoronalView and DICOMSagittalView
- Reconstruct ROI from Axial view to Sagittal and Coronal views
- Optimise the roi_display() function, from re-generating the dict_polygon everytime a structure box is checked/unchecked, to only add/remove polygon item that is checked/unchecked.

Please note that the ROI display is not perfect for a dataset that's lacking slice "thickness" (ie RT-02), but works on dataset with more slices like RT-05. The appearance of these ROIs is to be improved in a future release. Isodoses display on coronal and sagittal views is not supported.

### CODE REVIEW

File: data/line&fill_configuration [hunk 1]
```diff
@@ -1,5 +1,5 @@
 2
-50
```
Comment (tdlinh2712) [inline data/line&fill_configuration:2]: @DuyTungHa please remove this change

File: src/Model/ROI.py [hunk 2]
```diff
@@ -476,6 +475,65 @@ def calculate_pixels(pixlut, contour, prone=False, feetfirst=False):
...
+
+    Parameters
+    ----------
+    contour : object
+    """
+    pixels = []
+    print("sag")
+    np_x = np.array(pixlut[0])
+    np_y = np.array(pixlut[1])
+    # print(np_x, np_y)
```
Comment (tdlinh2712) [inline src/Model/ROI.py:496]: remove print in line 493 and 496 @DuyTungHa

**Review by tdlinh2712 — COMMENTED** [review]
@DuyTungHa

**Review by JThomas2020 — APPROVED** [review]
Team 19

**Comment by DuyTungHa** [comment]
@tdlinh2712 Done.

**Review by Tamookk — APPROVED** [review]
Team 14

**Pull request merged**